docs/gambit/mutation-verifier.md

```diff
@@ -112,9 +110,16 @@ We support `"staging" : true` as an alternative to `"staging" : "master"`.
 Omitting this key will cause verification to run locally
   (unless the run script has it).
 - `"cloud"` : if you instead want to run on the cloud environment you can provide the `--cloud` flag. You can also add the name of a specific branch.
-- `"use_cli_certora_run"` : Use CLI `certoraRun` rather than `certoraRun.py`. Expects a boolean and defaults to `false`.
+- `"use_certora_run_py"` : Use `certoraRun.py` rather than `certoraRun`. Expects a boolean and defaults to `false`.
 
 
+### Troubleshooting
+
+At the moment, there are a few ways in which `certoraMutate` can fail. Here we have some suggestions on how to troubleshoot when that happens. We are actively working on mitigating them.
```

Comment:
At the moment, there are a few ways in which `certoraMutate` can fail. Here, we have some suggestions on how to troubleshoot when that happens. We are actively working on mitigating them.